.github/workflows/ecs-deploy-v1.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           terraform_version: ${{ inputs.terraform_version }}
 
       - name: Terraform init
-        run: terraform init
+        run: terraform init -input=false
```

Comment:
i didn't know this thing could be interactive....
we have this in all our workflows

Reply:
the only blocking case we've met so far (twice) is when the target workspace doesn't exist